### ISSUE TITLE
Missing field `withdrawals` on Zhejiang

### ISSUE THREAD

**Issue by peteriscan** [ledgerwatch/erigon#6817]
#### System information

Erigon version: 2.38.0-dev-69e8ac0e

OS & Version: Ubuntu Linux 22.04.1 LTS, Docker version 23.0.0

Commit hash: 69e8ac0e774e5a7555ad6f1e0d431785fff712c0

Erigon Command (with flags/config): see below

Concensus Layer: Lighthouse

Concensus Layer Command (with flags/config): see below

Chain/Network: Zhejiang

#### Expected behaviour

Lighthouse returns a valid response for slot 43360.

#### Actual behaviour

Lighthouse returns an unhandled error because it can't parse the execution client payload.

```
$ curl http://localhost:5052/eth/v2/beacon/blocks/43360
{
  "code": 500,
  "message": "UNHANDLED_ERROR: ExecutionLayerErrorPayloadReconstruction(0xc3d5a8bd379f36c54e6ed0961258fc691726f4dd09cdb34839ff2b250ee1e95c, EngineError(Api { error: Json(Error(\"missing field `withdrawals`\", line: 0, column: 0)) }))",
  "stacktraces": []
}
```

#### Steps to reproduce the behaviour

Ubuntu 22.04.1 LTS

Install Docker
```
sudo apt-get install git curl jq
curl -fsSL get.docker.com | bash
```

Create datadirs
```
mkdir -p erigon lighthouse
sudo chown 1000 erigon lighthouse
echo 0xabcbeef13b8093560da4762facfd201ace798eee90b260bd941f3e904b6d4123 > jwt.hex
```

Zhejiang testnet data
```
git clone https://github.com/ethpandaops/withdrawals-testnet.git
```

Initialize Erigon database
```
sudo docker run --rm -it \
  -v $(pwd)/erigon:/data \
  -v $(pwd)/withdrawals-testnet/zhejiang-testnet/custom_config_data:/config:ro \
  thorax/erigon:devel \
  erigon init --datadir=/data /config/genesis.json
```

Run Erigon
```
sudo docker run --rm -it \
  --network host \
  -v $(pwd)/erigon:/data \
  -v $(pwd)/jwt.hex:/var/run/secrets/jwt.hex:ro \
  thorax/erigon:devel \
  erigon \
  --datadir=/data \
  --ethash.dagdir=/data/ethash \
  --clique.datadir=/data/clique \
  --authrpc.addr=0.0.0.0 \
  --authrpc.port=8551 \
  --authrpc.vhosts=* \
  --externalcl \
  --http \
  --http.addr=0.0.0.0 \
  --http.port=8545 \
  --http.api=eth,erigon,web3,net,debug,trace,txpool,db,engine \
  --http.vhosts=* \
  --http.compression \
  --authrpc.jwtsecret=/var/run/secrets/jwt.hex \
  --txpool.disable \
  --snapshots=false \
  --networkid=1337803 \
  --bootnodes=enode://691c66d0ce351633b2ef8b4e4ef7db9966915ca0937415bd2b408df22923f274873b4d4438929e029a13a680140223dcf701cabe22df7d8870044321022dfefa@64.225.78.1:30303,enode://89347b9461727ee1849256d78e84d5c86cc3b4c6c5347650093982b726d71f3d08027e280b399b7b6604ceeda863283dcfe1a01e93728b4883114e9f8c7cc8ef@146.190.238.212:30303,enode://c2892072efe247f21ed7ebea6637ade38512a0ae7c5cffa1bf0786d5e3be1e7f40ff71252a21b36aa9de54e49edbcfc6962a98032adadfa29c8524262e484ad3@165.232.84.160:30303,enode://71e862580d3177a99e9837bd9e9c13c83bde63d3dba1d5cea18e89eb2a17786bbd47a8e7ae690e4d29763b55c205af13965efcaf6105d58e118a5a8ed2b0f6d0@68.183.13.170:30303,enode://2f6cf7f774e4507e7c1b70815f9c0ccd6515ee1170c991ce3137002c6ba9c671af38920f5b8ab8a215b62b3b50388030548f1d826cb6c2b30c0f59472804a045@161.35.147.98:30303
```

Run Lighthouse
```
sudo docker run --rm -it \
  --network host \
  -v $(pwd)/lighthouse:/data \
  -v $(pwd)/withdrawals-testnet/zhejiang-testnet/custom_config_data:/config:ro \
  -v $(pwd)/jwt.hex:/var/run/secrets/jwt.hex:ro \
  sigp/lighthouse:capella \
  lighthouse \
    --testnet-dir /config \
    --datadir /data \
    beacon \
    --http-allow-sync-stalled \
    --execution-endpoint http://localhost:8551 \
    --execution-jwt /var/run/secrets/jwt.hex \
    --http \
    --http-address 0.0.0.0 \
    --reconstruct-historic-states \
    --slots-per-restore-point 256 \
    --boot-nodes enr:-Iq4QMCTfIMXnow27baRUb35Q8iiFHSIDBJh6hQM5Axohhf4b6Kr_cOCu0htQ5WvVqKvFgY28893DHAg8gnBAXsAVqmGAX53x8JggmlkgnY0gmlwhLKAlv6Jc2VjcDI1NmsxoQK6S-Cii_KmfFdUJL2TANL3ksaKUnNXvTCv1tLwXs0QgIN1ZHCCIyk,enr:-Ly4QOS00hvPDddEcCpwA1cMykWNdJUK50AjbRgbLZ9FLPyBa78i0NwsQZLSV67elpJU71L1Pt9yqVmE1C6XeSI-LV8Bh2F0dG5ldHOIAAAAAAAAAACEZXRoMpDuKNezAAAAckYFAAAAAAAAgmlkgnY0gmlwhEDhTgGJc2VjcDI1NmsxoQIgMUMFvJGlr8dI1TEQy-K78u2TJE2rWvah9nGqLQCEGohzeW5jbmV0cwCDdGNwgiMog3VkcIIjKA,enr:-MK4QMlRAwM7E8YBo6fqP7M2IWrjFHP35uC4pWIttUioZWOiaTl5zgZF2OwSxswTQwpiVCnj4n56bhy4NJVHSe682VWGAYYDHkp4h2F0dG5ldHOIAAAAAAAAAACEZXRoMpDuKNezAAAAckYFAAAAAAAAgmlkgnY0gmlwhJK-7tSJc2VjcDI1NmsxoQLDq7LlsXIXAoJXPt7rqf6CES1Q40xPw2yW0RQ-Ly5S1YhzeW5jbmV0cwCDdGNwgiMog3VkcIIjKA,enr:-MS4QCgiQisRxtzXKlBqq_LN1CRUSGIpDKO4e2hLQsffp0BrC3A7-8F6kxHYtATnzcrsVOr8gnwmBnHYTFvE9UmT-0EHh2F0dG5ldHOIAAAAAAAAAACEZXRoMpDuKNezAAAAckYFAAAAAAAAgmlkgnY0gmlwhKXoVKCJc2VjcDI1NmsxoQK6J-uvOXMf44iIlilx1uPWGRrrTntjLEFR2u-lHcHofIhzeW5jbmV0c4gAAAAAAAAAAIN0Y3CCIyiDdWRwgiMo,enr:-LK4QOQd-elgl_-dcSoUyHDbxBFNgQ687lzcKJiSBtpCyPQ0DinWSd2PKdJ4FHMkVLWD-oOquXPKSMtyoKpI0-Wo_38Bh2F0dG5ldHOIAAAAAAAAAACEZXRoMpDuKNezAAAAckYFAAAAAAAAgmlkgnY0gmlwhES3DaqJc2VjcDI1NmsxoQNIf37JZx-Lc8pnfDwURcHUqLbIEZ1RoxjZuBRtEODseYN0Y3CCIyiDdWRwgiMo,enr:-KG4QLNORYXUK76RPDI4rIVAqX__zSkc5AqMcwAketVzN9YNE8FHSu1im3qJTIeuwqI5JN5SPVsiX7L9nWXgWLRUf6sDhGV0aDKQ7ijXswAAAHJGBQAAAAAAAIJpZIJ2NIJpcIShI5NiiXNlY3AyNTZrMaECpA_KefrVAueFWiLLDZKQPPVOxMuxGogPrI474FaS-x2DdGNwgiMog3VkcIIjKA
```

Works
```
$ curl http://localhost:5052/eth/v2/beacon/blocks/43361 -s | jq | head
{
  "version": "capella",
  "execution_optimistic": false,
  "data": {
    "message": {
      "slot": "43361",
```

Doesn't work
```
$ curl http://localhost:5052/eth/v2/beacon/blocks/43360 -s | jq | head
{
  "code": 500,
  "message": "UNHANDLED_ERROR: ExecutionLayerErrorPayloadReconstruction(0xc3d5a8bd379f36c54e6ed0961258fc691726f4dd09cdb34839ff2b250ee1e95c, EngineError(Api { error: Json(Error(\"missing field `withdrawals`\", line: 0, column: 0)) }))",
  "stacktraces": []
}
```

Other slots that don't work: 43202, 43217, 43231

**Comment by fafrd** [comment]
I'm syncing up a node for zhejiang and then i'll take a look at this.

**Comment by fafrd** [comment]
Fixed by #6817. Requires a full resync

**Comment by peteriscan** [comment]
After resyncing with e31678f3557e8d035c1e4a8209e08e025f08ab8c Lighthouse no longer returns an error:

```
$ curl http://localhost:5052/eth/v2/beacon/blocks/43360 -s | jq | head                                                                                        
{
  "version": "capella",
  "execution_optimistic": false,
  "data": {
    "message": {
      "slot": "43360",
```

Thanks!
